Amplify/Categories/DataStore/Model/Embedded.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+// MARK: - CustomCodable
+
+/// All persistent custom types should conform to the CustomCodable protocol.
```

Comment:
update description.

also, maybe it would be good to expand on this a bit and explain the different between `Embedded` and `Model`